sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/ClientConstants.java

```diff
@@ -7,6 +7,7 @@
 
 public final class ClientConstants {
     public static final String AZURE_ACTIVE_DIRECTORY_SCOPE = "https://eventhubs.azure.net/.default";
+    public static final String AZ_NAMESPACE_VALUE = "Microsoft.EventHub";
```

Comment:
nit: Can you add a comment/link to where this value came from so when we visit this in the future, we'll know where it was from.